docs/publishing/submit-your-plugin.md

```diff
@@ -52,6 +52,7 @@ In this step, you'll submit your plugin to the Obsidian team for review.
    ```
 
    - `id`, `name`, `author`, and `description` determines how your plugin appears to the user, and should match the corresponding properties in your [plugin manifest](manifest-reference.md).
+   - `id` should be unique to your plugin and unique in the `community-plugins.json`. Therefore do a search to confirm there is no conflict.
```

Comment:
```suggestion
   - `id` is unique to your plugin. Search `community-plugins.json` to confirm that there's no existing plugin with the same id.
```

- "Should" implies that there are cases where it's ok.
- "Do a search" is a [nominalization](https://en.wikipedia.org/wiki/Nominalization), prefer "Search".